modules/ROOT/Edits/Configuring and Managing Message Flows/undeploy-message-flows.adoc

```diff
@@ -0,0 +1,23 @@
+== Undeploying Message Flows
+
+Undeploying a message flow prevents future messages from being processed and enables you to recover resources that are consumed by a message flow that you no longer need.
+
+You can undeploy your message flow from either a sandbox or a production environment. The undeployment process takes about five minutes, during which actions such as `Deploy` and `Delete` are unavailable.
```

Comment:
I wonder if we should specify here that you typically undeploy from sandbox to redeploy to production or to fixed errors encountered during testing?